src/lib/mina_lib/mina_lib.ml

```diff
@@ -902,6 +908,14 @@ let add_full_transactions t user_command =
   |> Deferred.don't_wait_for ;
   Ivar.read result_ivar
 
+let add_snapp_transactions t (snapp_parties : Parties.t list) =
```

Comment:
rename `snapp_parties` to `snapp_txns` ? to avoid confusion since it could mean all the parties in a single transaction

Reply:
Done.